src/taglibs/migrate/all-tags/dynamic-attributes.js

```diff
@@ -0,0 +1,19 @@
+module.exports = function migrate(el, context) {
+    el.forEachAttribute(attr => {
+        const name = attr.name;
+        const value = attr.value;
+        if (
+            name == undefined &&
+            (value.type == "MemberExpression" || value.type == "Identifier")
+        ) {
+            context.deprecate(
+                'The "${attributes}" is deprecated. Please use "...attributes" modifier instead. See: https://github.com/marko-js/marko/wiki/Deprecation:-w-*-Atrributes'
+            );
+            attr.spread = true;
+            if (value.property && value.property.value == "*") {
```

Comment:
This check should not be needed, to should be safe to just remove the existing dynamic attribute and add a new spread attribute like so:

```js
el.removeAttribute(attrName);
el.addAttribute({ spread: true, value: value });
```